student.rb

```diff
@@ -1,12 +1,12 @@
 require_relative './person'
 
 class Student < Person
-    def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
-      super(id, age, name, parent_permission: parent_permission)
-      @classroom = classroom
-    end
-  
-    def play_hooky
-      "¯\(ツ)/¯"
-    end
-  end
\ No newline at end of file
+  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] (OPTIONAL) As the previous reviewer requested, the `id` should not be passed as a parameter. Kindly rectify this issue 👍